File: bronze/14626/c++/main.cpp
#include <iostream>

int main()
{
    char isbn[14];
    std::cin >> isbn;

    int sum = 0, k;
    for (int i = 0; i < 13; i++) {

        if (isbn[i] == '*') {
            k = (i % 2 == 0) ? 1 : 3;
        } else {
            if (i % 2 == 0) {
                sum += (static_cast<int>(isbn[i]) - 48); 
            } else {
                sum += ((static_cast<int>(isbn[i]) - 48) * 3);
            }
        }
    }

    for (int i = 0; i < 10; i ++) {
        if ((sum + (i * k)) % 10 == 0) {
            std::cout << i;
            break;
        }
    }

    return 0;
}

File: bronze/14626/python/main.py

if (__name__ == "__main__"):
    isbn = input()
    sum = 0
    for i in range(13):
        if (isbn[i] == '*'):
            k = 1 if (i % 2 == 0) else 3
        else:
            sum += (int(isbn[i]) * (1 if (i % 2 == 0) else 3))
    
    for i in range(10):
        if ((sum + (i * k)) % 10 == 0):
            print(i)